app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+class SessionsController < ApplicationController
+  def create
+    @user = User.find_by(email: session_params[:email])
+
+    if @user&.authenticate(session_params[:password])
+      login!
+      render json: {
```

Comment:
Do you want to redirect the user based on the outcome of this function?

Reply:
Fixed now, but I wasn't really focusing much more on the logging in and out until the tickets that capture that more

Reply:
I can see what you mean, this is another good example of why we don't future code. It often results in incomplete code and it's difficult to know that changes are still needed. 
This ticket does deal with logging in and the subsequent redirect so I think that it's appropriate to include a redirect within the login function.

Reply:
Completely understandable, I'll keep that in mind, it should be added in the last push